core/client/app/mixins/editor-base-controller.js

```diff
@@ -212,14 +212,23 @@ export default Mixin.create({
     // TODO: Update for new notification click-action API
     showSaveNotification(prevStatus, status, delay) {
         let message = this.messageMap.success.post[prevStatus][status];
-        let path = this.get('model.absoluteUrl');
-        let type = this.get('postOrPage');
         let notifications = this.get('notifications');
+        let target;
```

Comment:
Just a style thing but we're trying to standardise on putting all non-assigning declarations on one line.
https://github.com/dockyard/styleguides/blob/master/engineering/javascript.md#variables